types/board_support_common.yaml

```diff
@@ -0,0 +1,121 @@
+      MREC_11/12/13 do not apply here
+      MREC_14_PilotFault: The control pilot voltage is out of range.
+      MREC_15_PowerLoss: The EVSE is unable to supply any power due to mains failure
+      MREC_16 does not apply here, that is for ISO protocol
+      MREC_17_EVSEContactorFault: Contactors fail to open or close on EVSE's side. May also include welding related errors.
+      MREC_18_CableOverTempDerate: Temperature of charging cable or connector assembly is too high, resulting in reduced power operation.
+      MREC_19_CableOverTempStop: Temperature of charging cable or connector assembly is too high, resulting in a stopped charging session.
+      MREC_20_PartialInsertion: Cable latch is raised due to incomplete insertion into the vehicle charging port.
+      MREC_21/22 does not apply here
+      MREC_23_ProximityFault: The proximity voltage is out of range.
+      MREC_24_ConnectorVoltageHigh: The output voltage of EVSE is high before charging starts or after charging ends.
+      MREC_25_BrokenLatch: The latch on the connector is broken.
+      MREC_26_CutCable: The output cable has been severed from the EVSE.
+    type: object
+    additionalProperties: false
+    required:
+      - event
+    properties:
+      event:
+        type: string
+        enum:
+          - 'A'
+          - 'B'
+          - 'C'
+          - 'D'
+          - 'E'
+          - 'F'
+          - 'PowerOn'
+          - 'PowerOff'
+          - 'ErrorDF'
+          - 'ErrorVentilationNotAvailable'
+          - 'ErrorBrownOut'
+          - 'ErrorEnergyManagement'
+          - 'PermanentFault'
+          - 'EvseReplugStarted'
+          - 'EvseReplugFinished'
+          - 'Disconnected'
+          - 'ConnectorLockCapNotCharged'
+          - 'ConnectorLockUnexpectedOpen'
+          - 'ConnectorLockUnexpectedClose'
+          - 'ConnectorLockFailedLock'
+          - 'ConnectorLockFailedUnlock'
+          - 'MREC_1_ConnectorLockFailure'
+          - 'MREC_2_GroundFailure'
+          - 'MREC_3_HighTemperature'
+          - 'MREC_4_OverCurrentFailure'
+          - 'MREC_5_OverVoltage'
+          - 'MREC_6_UnderVoltage'
+          - 'MREC_8_EmergencyStop'
+          - 'MREC_10_InvalidVehicleMode'
+          - 'MREC_14_PilotFault'
+          - 'MREC_15_PowerLoss'
+          - 'MREC_17_EVSEContactorFault'
+          - 'MREC_18_CableOverTempDerate'
+          - 'MREC_19_CableOverTempStop'
+          - 'MREC_20_PartialInsertion'
+          - 'MREC_23_ProximityFault'
+          - 'MREC_24_ConnectorVoltageHigh'
+          - 'MREC_25_BrokenLatch'
+          - 'MREC_26_CutCable'
+  ProximityPilot:
+    description: Current capability (ampacity) of the cable
+    type: object
+    additionalProperties: false
+    required:
+      - ampacity
+    properties:
+      ampacity:
+        description: Ampacity value of the cable assembly
+        type: string
+        enum:
+          - None
+          - A_13
+          - A_20
+          - A_32
+          - A_63
```

Comment:
A_70 missing?

Reply:
Or rename to A_63_70? It is the same resistor value

Reply:
or A_63_3ph_70_1ph?

Reply:
Yes good idea.. I think I A_63_70 is enough

Reply:
implemented